tests/functional/base.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (C) 2017  Custodia Project Contributors - see LICENSE file
+from __future__ import absolute_import
+
+import os
+import shutil
+import socket
+import subprocess
+import sys
+import time
+from string import Template
+
+import pytest
+
+from custodia.client import CustodiaHTTPClient
+from custodia.compat import url_escape
+from custodia.server.args import parse_args
+from custodia.server.config import parse_config
+
+
+CONFIGURATION = u"""
```

Comment:
so, I do not like the idea to have this configuration data on this file, this base file should be concentrated in the reusable methods and the Custodia server initialization. This configuration looks like specific for some tests but I'll not be enough for other cases, so I'd rather have this in the test file.